cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp

```diff
@@ -132,9 +132,12 @@
 (defun send-cram-set-perception-result (action-inst res)
   (send-prolog-query-1 (create-query "cram_set_perception_result" (list action-inst res))))
 
-(defun export-log-to-owl (filename)
+(defun export-log-to-owl (&optional (filename (concatenate 'string "default_log_" (write-to-string (truncate (cram-utilities:current-timestamp))) ".owl")))
   (send-prolog-query-1 (create-query "rdf_save" (list (concatenate 'string "\\'/home/ros/user_data/" filename "\\'" ) "[graph(\\'LoggingGraph\\')]"))))
 
+(defun export-belief-state-to-owl (&optional (filename (concatenate 'string "default_belief_state_" (write-to-string (truncate (cram-utilities:current-timestamp))) ".owl")))
+  (json-prolog:prolog-simple-1 (concatenate 'string "rdf_save(" (concatenate 'string "'/home/ros/user_data/" filename "'" "," "[graph('belief-state')])"))))
```

Comment:
Hey, could you please use "/home/ease/logs" as the directory? There is no "ros/user_data" directory on the robot, cuz this belief-state is stored locally.

Reply:
@seba90

Reply:
I fixed the store path for the belief state